app/cmd/restore_to_file.go

```diff
@@ -198,7 +200,8 @@ func CheckBackingFileFormat(backingFilePath string) error {
 }
 
 func CopyFile(backingFilepath, outputFile string) error {
-	if _, err := iutil.ExecuteWithoutTimeout("cp", []string{backingFilepath, outputFile}); err != nil {
+	exec := lhutils.NewExecutor().Execute
+	if _, err := exec(nil, "cp", []string{backingFilepath, outputFile}, lhtypes.ExecuteNoTimeout); err != nil {
```

Comment:
NIT: Do we need this function. It seems that we can use `lhns.CopyFiles` instead

Reply:
Why do we need the namespace switching for this? IIRC it's just a normal command execution.